querydsl-core/src/test/java/com/querydsl/core/support/ConstantHidingExpressionTest.java

```diff
@@ -0,0 +1,31 @@
+package com.querydsl.core.support;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import com.querydsl.core.Tuple;
+import com.querydsl.core.types.FactoryExpression;
+import com.querydsl.core.types.Projections;
+import com.querydsl.core.types.dsl.Expressions;
+
+public class ConstantHidingExpressionTest {
+
+    @Test
+    public void constant_removal() {
```

Comment:
To address @robertandrewbain's [comment](https://github.com/querydsl/querydsl/commit/ba39fe195ee4#commitcomment-14995908), this could be `constants_hidden`?